begrun.c

```diff
@@ -213,13 +217,16 @@ set_units(void)
     message(0, "UnitDensity_in_cgs = %g \n", All.UnitDensity_in_cgs);
     message(0, "UnitEnergy_in_cgs = %g \n", All.UnitEnergy_in_cgs);
     message(0, "Photon density OmegaG = %g\n",All.CP.OmegaG);
-    message(0, "Massless Neutrino density OmegaNu0 = %g\n",All.CP.OmegaNu0);
+    if(!All.MassiveNuLinRespOn)
+        message(0, "Massless Neutrino density OmegaNu0 = %g\n",get_omega_nu(&All.CP.ONu, 1));
```

Comment:
In class they call this omega_ur, ultra relativistic species. A reason it is cleaner is because it is hard to distinguish nu,  massive nu and masslessnu. What convention are we using here? nu is massless and mnu is massive?

Reply:
The convention here is that all neutrino masses are computed by get_omega_nu, whether massless or massive. There is no split between massless and massive neutrinos at all. MassiveNuLinRespOn controls only whether the neutrinos are supposed to be a component of Omega0.